Progress Watchdog, hardcoded default Namespace

https://github.com/iteratec/multi-juicer/blob/master/progress-watchdog/main.go#L78 there is the "default" Namespace hardcoded which does not work if deployed to any other namespace than "default".
 
Therefore I also see permission errors (as sa, role, rolebinding in an other namespace):

```
panic: deployments.apps is forbidden: User "system:serviceaccount:juicy-ctf:progress-watchdog" cannot list resource "deployments" in API group "apps" in the namespace "default"
goroutine 1 [running]:
main.createProgressUpdateJobs(0xc000180420, 0xc000271760)
/src/main.go:80 +0x5da
main.main()
/src/main.go:66 +0x2ff
```

--- a/progress-watchdog/main.go
+++ b/progress-watchdog/main.go
@@ -75,7 +75,8 @@ func createProgressUpdateJobs(progressUpdateJobs chan<- ProgressUpdateJobs, clie
 			LabelSelector: "app=juice-shop",
 		}
 
-		juiceShops, err := clientset.AppsV1().Deployments("default").List(opts)
+		namespace := os.Getenv("NAMESPACE")
+		juiceShops, err := clientset.AppsV1().Deployments(namespace).List(opts)
 		if err != nil {
 			panic(err.Error())
 		}
@@ -234,7 +235,8 @@ func cacheContinueCode(clientset *kubernetes.Clientset, teamname string, continu
 		panic("Could not encode json, to update continueCode and challengeSolved count on deployment")
 	}
 
-	_, err = clientset.AppsV1().Deployments("default").Patch(fmt.Sprintf("t-%s-juiceshop", teamname), types.MergePatchType, jsonBytes)
+	namespace := os.Getenv("NAMESPACE")
+	_, err = clientset.AppsV1().Deployments(namespace).Patch(fmt.Sprintf("t-%s-juiceshop", teamname), types.MergePatchType, jsonBytes)
 	if err != nil {
 		log.Errorf("Failed to path new continue code into deployment for team %s", teamname)
 		log.Error(err)

